src/view.rs

```diff
@@ -859,3 +859,23 @@ mod view_response_builder_tests {
         assert_eq!(expected, got);
     }
 }
+
+pub struct AllDocumentsViewValue {
```

Comment:
`_all_docs` is a view that emits `{ rev: doc._rev }`; by default, it does not include the actual documents, that's what `include_docs` is for. Since the response is identical to that of a view, `ViewResponse` can definitely be reused. Now, the request parameters do differ slightly from a view (although I'm not sure why, maybe because `_all_docs` with `attachments` would be extremely expensive on a larger database), so a separate action type is probably better.